src/main/java/com/spartronics4915/frc2020/RobotContainer.java

```diff
@@ -220,6 +224,7 @@ public Command getAutonomousCommand()
             intermediate[i] = new Pose2d(pose.getTranslation().getX() - Units.inchesToMeters(312.5),
                     pose.getTranslation().getY(), pose.getRotation());
         }
+        System.out.println(mStateEstimator);
```

Comment:
is there a reason to not use Logger here?  Is it possible this is a spurious print?